Docs 7161 edit (#602)
mulesoft/docs-connectors: modules/ROOT/pages/email/email-trigger.adoc

The example above only dispatches email messages that contain the word
"IMPORTANT" in their subject.

> == Naming Strategy For Attachments

Reviewer: ```suggestion
== Naming Strategy for Attachments
```

> Email Connector provides an Attachment Naming Strategy parameter that enables you to specify a strategy for how to name attachments for the List and On New Email operations. 

Reviewer: ditto prev. rewording suggestions

This parameter is set at the configuration level, but can be overridden at the operation level. 

You can specify one of the following values for the parameter:

* `NAME` (default) + 
When the value is set to `NAME`, the connector names the attachment after the value of the `filename` field in the `Content-Disposition` line of the attachment (this is the standard and recommended way of setting the name for an attachment). If the `filename` field is not present or is empty, then the attachment is named "Unnamed".
If multiple attachments have no name then they are called "Unnamed", "Unnamed_1", "Unnamed_2", and so on so that they can be addressed individually.
* `NAME_HEADERS` + 

> When the value is set to `NAME_HEADERS`, the connector first tries to obtain the name from the `filename` field in the `Content-Disposition` (the same as when using `NAME`). If that fails, the connector looks for the `name` header in the `Content-Type` line of the attachment. If the `name` header is not present or is empty, then the attachment is named "Unnamed".

Reviewer: ```suggestion
When the value is set to `NAME_HEADERS`, the connector first tries to obtain the name from the `filename` field in the `Content-Disposition` line (the same as when using `NAME`). If that fails, the connector looks for the `name` header in the `Content-Type` line of the attachment. If the `name` header is not present or is empty, then the attachment is named "Unnamed".
```

* `NAME_HEADERS_SUBJECT` +

> When the value is set to `NAME_HEADERS_SUBJECT`, the connector tries the previous strategies (first it looks for the `filename` field and then for the `name` header). If both fail, then the connector checks if the attachment itself is an email (a nested message) and looks for the `Subject` of the email. If the subject is present and is not empty, then the attachment is named after the subject. Otherwise, the attachment is named "Unnamed".

Reviewer: Do we have to describe the previous two strategies?

Reviewer: ```suggestion
When the value is set to `NAME_HEADERS_SUBJECT`, the connector tries the two previously described strategies. If both fail, then the connector checks if the attachment itself is an email (a nested message) and looks for the subject of the email. If the subject is not empty, then the attachment is named after the subject. Otherwise, the attachment is named "Unnamed".
```
